query_optimizer/resolver/Resolver.cpp

```diff
@@ -1032,7 +1032,7 @@ L::LogicalPtr Resolver::resolveSelect(
       L::LogicalPtr sorted_logical_plan = resolveSortInWindow(logical_plan,
                                                               resolved_window);
 
-      WindowPlan window_plan(resolved_window, sorted_logical_plan);
+      WindowPlan window_plan(std::move(resolved_window), sorted_logical_plan);
 
       sorted_window_map.emplace(window.name()->value(), window_plan);
```

Comment:
Please use `move` in `sorted_window_map` as well:

`sorted_window_map.emplace(window.name()->value(), std::move(window_plan));`